tests/rt.sh

```diff
@@ -413,7 +413,7 @@ if [[ $TESTS_FILE =~ '35d' ]]; then
   TEST_35D=true
 fi
 
-BL_DATE=20210419
+BL_DATE=20210421
```

Comment:
I remember this PR will only change results with the 32bit build, is it still the case?

Reply:
Yes. See this file that lists all the files changed compared to the current baseline (20210419):

[develop-20210419-20210421.txt](https://github.com/ufs-community/ufs-weather-model/files/6355093/develop-20210419-20210421.txt)

Basically only 32bit tests, and only 32bit tests, are changed.